lib/runner.js

```diff
@@ -55,6 +57,10 @@ optionChain(chainableMethods, function (opts, title, fn) {
 		opts.serial = true;
 	}
 
+	if (this._match) {
```

Comment:
Once option forwarding is fixed you'll always have a value here. However you must make sure to only run the next line if `this._match` is not empty. By default it will be.